mods/ra/rules/aircraft.yaml

```diff
@@ -97,9 +97,16 @@ MIG:
 		HP: 70
 	Armor:
 		Type: Light
+	RevealsShroud@GROUNDED:
```

Comment:
Because 3 out of 5 RA planes are not supposed to have any sight at all (both badgers and the Spy Plane), having to disable all these traits on those 3 would be uglier than defining them on the MiG and Yak directly, in my opinion.